server/views/admin-list-categories.hbs

```diff
@@ -1,33 +1,47 @@
-<div class="container">
-     
-      <div class="row justify-content-md-center">
-        {{#each category}}
-        {{#if this.visible}}
-        <a href="/admin/categories/edit/{{this._id}}">
-            <div class=" card" style="width: 20rem;">
-                <div class="card-block">
-                    <h4 class="card-title">Category:{{this.title}}</h4>
-                     <p class="card-text">{{this.shortDescription}}</p>
-                </div>
-            </div>
-        </a>
-        {{else}}
-        <a href="/admin/categories/edit/{{this._id}}">
-            <div class="card  bg-success text-danger" style="width: 20rem;">
-                <div class="card-block">
-                   <p>This category is hide </p>
-                    <h4 class="card-title">Category:{{this.title}}</h4>
-                     <p class="card-text">{{this.shortDescription}}</p>
-                </div>
-            </div>
-        </a>
-         {{/if}}
-        {{/each}} 
-</div>
-     </div>
-    <a href="categories/add">
-        <button class="bg-info"> Add New category</button>
+    <a  href="categories/add">
+        <button class="btn btn-success addNewArticle"> Add New category</button>
     </a>
 </div>
 
+ 
+ <table class="table table-bordered table-danger">
+  <thead class="thead-inverse">
+    <tr>
+       <th>ID</th> 
+      <th>Title</th>
+      <th>Short content</th>
+        <th>Visible</th>
+      <th>Update</th>
+
+    </tr>
+  </thead>
+  <tbody>
+     {{#each category}}
+
+    {{#if this.visible}}
+
+    <tr>
+      
+      <th scope="row">{{this._id}}</th>
+      <td> {{this.title}}</td>
+      <td>{{this.shortDescription}} </td>
+      <td>{{this.visible}} </td>
+
+      <td><a href="/admin/categories/edit/{{this._id}}"><button class="btn btn-info">Edit</button> </a> </td>
+        
+     </tr> 
+      {{else}}
+      <tr>
+      
+      <th scope="row">{{this._id}}</th>
+      <td> {{this.title}}</td>
+      <td> <strike>hide</strike></td>
```

Comment:
why do you "hide" here - just put the shortDescription.
also `strike` the title - all of them (title, description, and visible`